ecs-agent/tmds/server.go

```diff
@@ -0,0 +1,155 @@
+// copyright amazon.com inc. or its affiliates. all rights reserved.
+//
+// licensed under the apache license, version 2.0 (the "license"). you may
+// not use this file except in compliance with the license. a copy of the
+// license is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. this file is distributed
+// on an "as is" basis, without warranties or conditions of any kind, either
+// express or implied. see the license for the specific language governing
+// permissions and limitations under the license.
+package tmds
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/didip/tollbooth"
```

Comment:
nit: i think we follow the following go imports structure as a best practice:

1st block for standard imports, 2nd block for amazon-ecs-agent imports, and the 3rd block for external imports

Reply:
Sure, I will fix this for all files.

Reply:
Updated